docs/development/docs-pipeline.md

```
@@ -0,0 +1,315 @@
+./builder docs:build     # gather -> index -> compile  (the whole site)
+./builder docs:dev       # gather-dev (symlinks) + docusaurus start, live reload
+./builder docs:serve     # preview the built site from dist/docs
+./builder docs:test      # unit tests for the docs helpers
+./builder docs:export    # refresh the generated copies under packages/ and .rocketride/
+./builder docs:check     # same computation, read-only: fail if a copy has drifted
+./builder docs:clean     # drop the content tree, dist/docs, .docusaurus, and the gather hash
+```
+
+`docs:build` expands to:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)